lib/ansible/executor/module_common.py

```diff
@@ -104,6 +126,9 @@
 # INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
 # LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+%(min_python)s
```

Comment:
Move this down below the existing check (`if sys.version_info < (3,):`) -- that will fix the macOS failures.